girder-dandi-archive/girder_dandi_archive/rest.py

```diff
@@ -326,3 +331,35 @@ def stats(self, params):
             "cell_count": cell_count,
             "size": drafts["size"],
         }
+
+    @access.user
+    @autoDescribeRoute(
+        Description("Lock a Dandiset").param("identifier", "Dandiset Identifier", paramType="path")
+    )
+    @dandiset_identifier
+    def lock_dandiset(self, identifier, params):
+        locking.lock(identifier, self.getCurrentUser())
+
+    @access.user
+    @autoDescribeRoute(
+        Description("Unlock a Dandiset").param(
+            "identifier", "Dandiset Identifier", paramType="path"
+        )
+    )
+    @dandiset_identifier
+    def unlock_dandiset(self, identifier, params):
+        locking.unlock(identifier, self.getCurrentUser())
+
+    @access.public
+    @autoDescribeRoute(
+        Description("The owner of the lock on a Dandiset").param(
+            "identifier", "Dandiset Identifier", paramType="path"
+        )
+    )
+    @dandiset_identifier
+    def get_dandiset_lock_owner(self, identifier, params):
+        owner = locking.get_lock_owner(identifier)
+        if owner is None:
+            return
+        print("lock owned by", owner)
```

Comment:
I'd prefer if the return was explicit here, and the print statement removed:

```suggestion
            return None
```

Reply:
and who sees those `print`s?

Reply:
This is leftover debug code, I will remove.